views/index.html

```diff
@@ -187,11 +187,13 @@ <h2 class="call-to-action">Let's Talk!</h2>
 
   <!-- This needs an additional class of dark-bg when 'What our clients say' gets added -->
   <div class="copyright-block">
+    <p>dwyl</p>
```

Comment:
@markwilliamfirth could we follow the "Address Microformat" here?
see: http://microformats.org/wiki/adr#Examples
This will make the address show up on google results and thus link directly to a google map when people search for us in their fav search engine. 😉